classes/Import.php

```diff
@@ -36,7 +36,25 @@ function array_key_first(array $arr)
 
 class Import
 {
-    const TRANSLATION_FILE_BEGINS = "<?php\nglobal \$_MODULE;\n\$_MODULE = array();\n";
+    const TRANSLATION_LICENSE = "/**
+ * 2007-2021 PrestaShop and Contributors
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Academic Free License 3.0 (AFL-3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/AFL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2019 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/AFL-3.0 Academic Free License 3.0 (AFL-3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */\n";
```

Comment:
Can you replace the header by this one? https://github.com/PrestaShopCorp/header-stamp/blob/master/assets/afl.txt

Thanks!

Reply:
Done